test/unit/model/publishing/test/TestBackupServiceTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+declare(strict_types=1);
+
+namespace oat\taoPublishing\test\unit\model\publishing\test;
+
+use Exception;
+use common_Exception;
+use core_kernel_classes_Property;
+use core_kernel_classes_Resource;
+use oat\generis\model\data\Ontology;
+use oat\generis\test\MockObject;
+use oat\oatbox\filesystem\File;
+use oat\oatbox\log\LoggerService;
+use oat\tao\model\import\service\RdsResourceNotFoundException;
+use oat\taoPublishing\model\publishing\test\TestBackupService;
+use oat\generis\test\TestCase;
+use oat\taoQtiTest\helpers\QtiPackageExporter;
+
+class TestBackupServiceTest extends TestCase
+{
+    /** @var TestBackupService */
+    private $subject;
+
+    /** @var Ontology|MockObject */
+    private $ontologyMock;
+
+    /** @var QtiPackageExporter|MockObject */
+    private $qtiPackageExporterMock;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->ontologyMock = $this->createMock(Ontology::class);
+        $this->qtiPackageExporterMock = $this->createMock(QtiPackageExporter::class);
+        $serviceLocatorMock = $this->getServiceLocatorMock(
+            [
+                Ontology::SERVICE_ID => $this->ontologyMock,
+                QtiPackageExporter::SERVICE_ID => $this->qtiPackageExporterMock,
+                LoggerService::SERVICE_ID => $this->createMock(LoggerService::class)
+            ]
+        );
+
+        $this->subject = new TestBackupService();
+        $this->subject->setServiceLocator($serviceLocatorMock);
+    }
+
+    public function testBackupDeliveryTestPackage_ThrowsExceptionWhenDestDoesNotExist(): void
```

Comment:
Dest?

Reply:
It was type. Removed with test case as it's not  needed any more with test resource available from event.